_topic_map.yml

```diff
@@ -2762,37 +2762,81 @@ Topics:
   - Name: Removing Jaeger
     File: rhbjaeger-removing
 ---
-Name: OpenShift Virtualization
+Name: OpenShift/OKD Virtualization
```

Comment:
I fear we have to solve this before bothering anybody to get an ack

Reply:
Ok, I'm on it.